roles/base/tasks/main.yml

```diff
@@ -38,13 +38,18 @@
 - name: install and start ntp
   shell: systemctl enable ntpd
   when: ansible_os_family == "RedHat"
+- name: check if golang is installed
+  stat: path=/usr/local/go/bin/go
+  register: golang_bin
 - name: download Golang v1.5.2
   get_url:
```

Comment:
@mapuri This file is downloaded in /tmp. It won't exist when we provision the VMs and we'll download it if we don't use stat. I've already looked into the other options.